### PR TITLE
Added API Key fallback and print statments

### PR DESCRIPTION
**Pull request by lunamidori5**
**Description**

This PR fixes api keys not being used

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

@mudler this is the pr from discord

### CODE REVIEW

File: core/startup/config_file_watcher.go [hunk 1]
```diff
@@ -15,18 +15,31 @@ import (
...
+
+    fileContent, err := os.ReadFile(path.Join(configDir, "api_keys.json"))
+    if err != nil {
+        // Check if the file exists in the /build directory
+        buildFilePath := path.Join("/build", "api_keys.json")
+        fileContent, err = os.ReadFile(buildFilePath)
+        if err != nil {
+            return err
+        } else {
+            fmt.Println("Loaded api_keys.json from /build") // Indicate successful load from /build
```
Comment (cryptk) [inline core/startup/config_file_watcher.go:28]: `log.Debug().Msg("Falling back to loading api_keys.json from /build")

File: core/startup/config_file_watcher.go [hunk 2]
```diff
@@ -15,18 +15,31 @@ import (
...
-		var fileKeys []string
-		err := json.Unmarshal(fileContent, &fileKeys)
-		if err == nil {
-			appConfig.ApiKeys = append(appConfig.ApiKeys, fileKeys...)
-			return nil
-		}
-		return err
-	}
-	return err
+    fmt.Println("Config directory:", configDir) // Print the configDir
```
Comment (cryptk) [inline core/startup/config_file_watcher.go:18]: don't use fmt.Println, use something like this instead:

`log.Debug().Str("config_dir", configDir).Msg("Configuration directory path")`

File: core/startup/config_file_watcher.go [hunk 3]
```diff
@@ -15,18 +15,31 @@ import (
...
+        // Check if the file exists in the /build directory
+        buildFilePath := path.Join("/build", "api_keys.json")
+        fileContent, err = os.ReadFile(buildFilePath)
+        if err != nil {
+            return err
+        } else {
+            fmt.Println("Loaded api_keys.json from /build") // Indicate successful load from /build
+        }
+    } else {
+        fmt.Println("Loaded api_keys.json from config directory") // Indicate successful load from configDir
```
Comment (cryptk) [inline core/startup/config_file_watcher.go:31]: log.Debug().Msg("Loaded api_keys.json from config directory")

**Review by cryptk — CHANGES_REQUESTED** [review]
please use the logging facilities rather than directly using fmt.Print/Println

File: core/startup/config_file_watcher.go (hunk 1 above)
Comment (lunamidori5) [reply core/startup/config_file_watcher.go:28]: is there a import or from or what ever needed added?

**Comment by lunamidori5** [comment]
> please use the logging facilities rather than directly using fmt.Print/Println

@cryptk I did this

**Pull request closed without merging**